pkg/util/cloudfoundry/types.go

```diff
@@ -0,0 +1,204 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build clusterchecks
+
+package cloudfoundry
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Comment:
Let's have external dependencies grouped together. This applies everywhere of course.